docs/source/release_notes/release-notes.rst

```diff
@@ -42,6 +42,15 @@ develop
     *    - Type
          - Change
 
+    *    - |devbreak|
+         - The implementation of the TimeLock server has been separated into a new project, `timelock-impl`.
+           (`Pull Request <https://github.com/palantir/atlasdb/pull/????>`__)
+
+    *    - |userbreak|
+         - `TimelockServerConfiguration` now requires an algorithm to be specified.
+           Previously, if no algorithm was specified, the configuration would default to atomix.
+           (`Pull Request <https://github.com/palantir/atlasdb/pull/????>`__)
+
```

Comment:
Also: this technically causes a user break for consumers of TimeLock metrics (because of the package name change).